roles/benchmarking/benchmarking_run_mlperf_ssd/tasks/main.yml

```diff
@@ -7,24 +7,24 @@
   command: oc get nodes -l kubernetes.io/hostname={{ benchmarking_run_mlperf_ssd_node_hostname }} -oname
 
 - name: Ensure that the coco dataset PVC exists
-  command:
-    oc get pvc/{{ benchmarking_run_mlperf_ssd_dataset_pvc_name }}
+  shell: |
+    oc get pvc/{{ benchmarking_run_mlperf_ssd_dataset_pvc_name }} \
```

Comment:
I'm not sure to understand why you turn this `command` into a `shell`? 🤔 
the way I usually proceed is that if there is nothing requiring bash (`|` or `> file`, most of the time), I use a `command`

Reply:
looks like a typo, it's the only time this happens 🙃 ;-)

Reply:
Done